src/mesh/simplemesh.jl

```diff
@@ -6,6 +6,9 @@
     SimpleMesh(points, connec)
 
 A simple mesh with `points` and connectivities `connec`.
+Note: please make sure all the connectivity polygons have
+`orientation(polygon) === :CCW`.
+
```

Comment:
Please follow the convention for notes in docstring in other docstrings. We have notes at the end of the docstring in a separate section. Also notes usually don't use code, they just describe the situation in simple English words.

Reply:
> This note has nothing to do with the next part, that's why I put it there.

As I explained all notes in our docstrings appear as the last section. You didn't understand that.

> CCW makes no sense in 3D, and the limitation might just be for 2D, but I'm not sure.

Coherent orientation makes sense in any dimension. The fact that we use the term "CCW" to refer to it is irrelevant.

> Giving up.

We have high standards, sorry.

Reply:
> Coherent orientation makes sense in any dimension. The fact that we use the term "CCW" to refer to it is irrelevant.

"CCW" has a very precise meaning, and in 3D it depends on some convention, that would need to be expressed.

> We have high standards, sorry.

That's good. It just became obvious that this domain is too foreign for me to contribute with quality.
Please keep on the good work as you see fit.